### PR TITLE
Fix bugs introduced by setup-sslmate

### PR DESCRIPTION
**Pull request by king-alexander**
#  #

## 🗣 Description ##

I introduced a few bugs to the project when I rebased #31. This small pull request fixes those mistakes.
## 💭 Motivation and context ##

These changes are required to remove my bugs from `develop`.

## 🧪 Testing ##

I corrected the inadvertent changes from rebasing #31.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Add a tag or create a release.

### CODE REVIEW

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Please fix these badges to refer to this repository:

https://github.com/cisagov/admiral/blob/9dd0abe5f80c08e77b47a50f8be60830add11cf3/README.md?plain=1#L3-L6

**Review by dav3r — APPROVED** [review]
👍

File: src/admiral/_version.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 """This file defines the version of this module."""
-__version__ = "0.2.0"
+__version__ = "1.1.0"
```
Comment (mcdonnnj) [inline src/admiral/_version.py]: Just a reminder to bump the minor version.

**Review by mcdonnnj — APPROVED** [review]

File: src/admiral/_version.py (hunk 1 above)
Comment (king-alexander) [reply src/admiral/_version.py]: [6d98461](https://github.com/cisagov/admiral/pull/39/commits/6d98461ba2412e0cc43f5428b63a688231faaa0c)

**Pull request merged**